payload_bay_micro.ino

```diff
@@ -30,7 +62,7 @@ void loop() {
   switch (mode) {
   case BELOW_5K:
```

Comment:
why are we not adding a point to the buffer to be sent to the radio in the "BELOW_5K" mode but we are in the other two? Wouldn't we want ot add a poin to the buffer to be sent to the radio in all modes?

Reply:
you're right, that's an oversight on my part.